.gitignore

```diff
@@ -45,93 +31,101 @@ tags
 cscope*
 
 # Stuff made by our makefiles
-*.pc
-*.log
 *.trs
 
-## The initial / makes these files only get ignored in particular directories.
-/autom4te.cache
-
 # configure in progress
+*.err
+*.tmp
 /.cyg*
 /confdefs.*
+/conftest
 /conftest.*
 
-# Libtool adds these, at least sometimes
+# files generated by autotools (`./autogen.sh`)
+## The initial / makes these files only get ignored in particular directories.
+/aclocal.m4
+/autom4te.cache
+/build-aux/
+/configure
+/config.h.in
 /m4/libtool.m4
 /m4/ltoptions.m4
 /m4/ltsugar.m4
 /m4/ltversion.m4
 /m4/lt~obsolete.m4
+/Makefile.in
 
-/aclocal.m4
-compile
-config.cache
-config.guess
+# files generated by configure (`mkdir build; cd build && ../configure`)
+*.log
+*.pc
+.deps/
+config.h
 config.log
 config.status
-config.sub
-configure
-configure.lineno
-depcomp
-config.h
-config.h.in
-install-sh
+evconfig-private.h
 libtool
-ltmain.sh
-missing
+Makefile
 stamp-h1
 stamp-h2
 
+# other generated config files (which command?)
+config.cache
+config.guess
+config.sub
+
 # files generated by doxygen
 doxygen
 CMakeDoxyfile.in
 CMakeDoxygenDefaults.cmake
 Doxyfile.doxygen
 
-# sample binaries
-dns-example
-event-read-fifo
-hello-world
-http-server
-http-connect
-le-proxy
-https-client
-https-client-mbedtls
-ssl-client-mbedtls
-signal-test
-time-test
-event-test
-watch-timing
-becat
-ws-chat-server
-
-# test binaries
-test-driver
-bench
-bench_cascade
-bench_http
-bench_httpclient
-regress
+# files generated by make (`mkdir build; cd build && ../configure && make`)
+.dirstamp
+.libs/
+*.la
+*.lo
+*.o
+event-config.h
+sample/becat
+sample/dns-example
+sample/event-read-fifo
+sample/event-test
+sample/hello-world
+sample/http-connect
+sample/http-server
+sample/https-client
+sample/https-client-mbedtls
+sample/le-proxy
+sample/signal-test
+sample/ssl-client-mbedtls
+sample/time-test
+sample/watch-timing
+sample/ws-chat-server
+test/bench
+test/bench_cascade
+test/bench_http
+test/bench_httpclient
+test/regress
+test/rpcgen-attempted
+test/test-changelist
+test/test-closed
+test/test-driver
+test/test-dumpevents
+test/test-eof
+test/test-fdleak
+test/test-init
+test/test-kq-collision
+test/test-ratelim
+test/test-script.sh
+test/test-time
+test/test-weof
```

Comment:
This should be reverted, otherwise:

```
mkdir bld
cd bld
../configure
make -j
git st -u
```

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        sample/becat
        sample/dns-example
        sample/event-read-fifo
        sample/hello-world
        sample/http-connect
        sample/http-server
        sample/https-client
        sample/https-client-mbedtls
        sample/le-proxy
        sample/signal-test
        sample/ssl-client-mbedtls
        sample/time-test
        sample/watch-timing
        sample/ws-chat-server
        test/bench
        test/bench_cascade
        test/bench_http
        test/bench_httpclient
        test/regress
        test/rpcgen-attempted
        test/test-changelist
        test/test-closed
        test/test-dumpevents
        test/test-eof
        test/test-fdleak
        test/test-init
        test/test-ratelim
        test/test-time
        test/test-weof

```

Reply:
I see. 2ea33f478edc1c7b9b0f3d1210a9bbffa02ca693